assets/javascripts/discourse/components/group-tracker-nav.gjs

```diff
@@ -135,7 +135,8 @@ export default class GroupTrackerNav extends Component {
           @action={{this.jumpToPrevTrackedPost}}
           @icon={{this.prevTrackerIcon}}
           @disabled={{this.prevTrackedPostDisabled}}
-          ariaLabel={{this.ariaLabelPrevPost}}
+          @ariaLabel={{this.ariaLabelPrevPost}}
+          @title={{this.ariaLabelPrevPost}}
```

Comment:
both `@ariaLabel` and `@title` arguments to d-button take a i18n id, not the translated string we're passing here. you'd have to use `@translatedAriaLabel` or `@translatedTitle`, or better yet (imo) override the attributes directly:

```suggestion
          aria-label={{this.ariaLabelPrevPost}}
          title={{this.ariaLabelPrevPost}}
```